### PR TITLE
Zm new version docs

### PR DESCRIPTION
**Pull request by zacharym**
@paetling @jsingle @thieman open to suggestions here

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -41,7 +41,21 @@ $ mkdocs serve
...
-* Bump the version number in dusty/constants.py to the *next* version
-* Bump the version of the curl command on the Readme, and in `docs/installation.md`
+* Update the changelog
+  * Add a date to the changelog for this version
+  * Add a new version that has an `In progress` title rather than a date
+  * Update the changelog to reflect your new changes with appropriate descriptions
+    * NEW: This change is an addition to the existing code/functionality, and should include a description of what the addition does.
+    * BREAKING: This update changes the existing promises of the dusty CLI API or specs API
+    * FIXED: This is a patch for something broken in the current dusty version
+    * other comments: anything that is not one of the above
```
Comment (paetling) [inline docs/contributing.md]: but still has big user impact

**Comment by paetling** [comment]
:ghost: :cow:

File: docs/contributing.md (hunk 1 above)
Comment (thieman) [inline docs/contributing.md]: Adding shit to the changelog isn't something you do when you cut a release, it's something you do when you merge new features and fixes and stuff

File: docs/contributing.md [hunk 2]
```diff
@@ -41,7 +41,21 @@ $ mkdocs serve
...
+  * Submit a pull request for your changelog updates and merge once accepted
+
+* Cut a new version of Dusty
+  * Run the `DustyRelease` Jenkins job with your new version number supplied as a parameter
+
+* Update README.md to reflect the new Dusty version
+  * After the `DustyRelease` Jenkins job has successfully run, change the curl target specified in the readme to reference the newly minted version
+  * Create a pull request for this change and merge once accepted
```
Comment (thieman) [inline docs/contributing.md]: Isn't this referenced in the Install docs somewhere too

**Pull request merged**